### PR TITLE
Added SOP for restarting master nodes

### PR DESCRIPTION
**Pull request by robshelly**
## What
SOP for restating master node instances in AWS

## Why
This SOP is needed when AWS retires EC2 instances which contain master nodes

## How
SOP added to sops directory

### CODE REVIEW

**Comment by StevenTobin** [comment]
Looks like you're missing the commits, this is just the template

File: sops/restart-master-nodes.asciidoc [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+. Get the provider ID
+
++
+```sh
+oc get machine <MACHINE_NAME> -n openshift-machine-api -o yaml | grep providerID
+```
+
+== Execute/Resolution
+
+. Login to AWS Console. See AWS.
```
Comment (StevenTobin) [inline sops/restart-master-nodes.asciidoc:63]: Is there supposed to be a link or something here?

File: sops/restart-master-nodes.asciidoc [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+. Navigate to EC2 in the correct region.
+. Select a master instance
+. Select "Tags" to verify the cluster name. Look for tag "kubernetes.io/cluster/": "owned"
+. Stop the master instance
+. Start the master instance
+. Wait for the instance to reach a running state
+
+== Validate
+
+. Verify the instanace is in an up and running state
```
Comment (StevenTobin) [inline sops/restart-master-nodes.asciidoc:73]: Typo

**Review by StevenTobin — CHANGES_REQUESTED** [review]

**Review by david-martin — APPROVED** [review]
These steps look reasonable.
@StevenTobin Are you happy with your requested changes have all been addressed?

**Review by StevenTobin — APPROVED** [review]

**Pull request merged**